posts/201805.md

```diff
@@ -26,13 +26,13 @@ Atomic swap with Ethereum on testnet was [demonstrated](https://twitter.com/bebr
 
 Two new faces were [added](https://github.com/decred/dcrweb/commit/9a410a33c2e3dc95405c264523d28c43678123e6) to [contributors](https://decred.org/contributors/) page.
 
-Dev activity stats for May: 238 active PRs, 195 master commits, 32,831 added and 22,280 deleted lines spread across 8 repositories done by 4-10 developers per repository. ([chart](https://twitter.com/decredproject/status/1002963653778214912))
+Dev activity stats for May: 238 active PRs, 195 master commits, 32,831 added and 22,280 deleted lines spread across 8 repositories. Contributions came from 4-10 developers per repository. ([chart](https://twitter.com/decredproject/status/1002963653778214912))
 
 ## Network
 
 Hashrate: rapid growth from ~4,000 TH/s at the beginning of the month to ~15,000 at the end with new all time high of 17,949. Interesting dynamic in [hashrate distribution](https://dcred.eu/powStats) across mining pools: coinmine.pl share down from 55% to 25% while F2Pool up from 2% to 44%.
 
-Staking: 30-day average ticket price is 91.3 DCR (+0.8), stake participation is 46.9% (+0.8%) with 3.68 million DCR locked (+0.15). Min price was 85.56. On May 11 ticket price surged to 96.99, staying elevated for longer than usual. Locked DCR peaked at 47.17%. This was [speculated](https://www.reddit.com/r/decred/comments/8ixay1/decred_investment_thesis_by_placeholder_ventures/dyvpgg5/?context=3) to be a result of a large miner staking.
+Staking: 30-day average ticket price is 91.3 DCR (+0.8), stake participation is 46.9% (+0.8%) with 3.68 million DCR locked (+0.15). Min price was 85.56. On May 11 ticket price surged to 96.99, staying elevated for longer than usual after such a pump. Locked DCR peaked at 47.17%. [nrpalao](https://www.reddit.com/user/nrpalao) on reddit [suggested](https://www.reddit.com/r/decred/comments/8ixay1/decred_investment_thesis_by_placeholder_ventures/dyvpgg5/?context=3) that the DCR for these tickets likely came from a miner with significant hashrate.
```

Comment:
I like the idea of crediting reddit user, but nrpalao submitted Decred Investment Thesis, while isrly_eder noticed the whale staking and jet_user elaborated on possible (but very likely) connection with large miner.

Reply:
Oops, got my names mixed up there.